### PR TITLE
Finish mobile porfolio setup for code review.

### PR DESCRIPTION
**Pull request by demix007**
# In this milestone for the finishing up of my mobile portfolio site, I:

- I created a new branch and named it "mobile_portfolio_finish_branch"

- I pulled all changes made from the feature branch.

- Edited the index.html file that contains the markup for the portfolio site, in order to replicate the figma template.

- Edited  style.css file that contains the styling sheet for the site also.

- I added all changes to the new branch, made a commit and push.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,65 +1,247 @@
...
+                                standard dummy text ever since the 1500s, when an unknown 
+                                printer took a standard dummy text.
+                            </p>
+                            <ul class="automation">
+                                <li class="css">css</li>
+                                <li class="html">html</li>
+                                <li class="bootstrap">bootstrap</li>
+                                <li class="ruby">Ruby</li>
+                            </ul>
+                            <button type="button" class="btn">See Project</button>
```
Comment (ichala) [inline index.html:89]: - [x] great job ! your website looks awesome and clean! kindly add the button styling: Active, Hover, and Pressed like shown in figma

image

File: index.html [hunk 2]
```diff
@@ -1,65 +1,247 @@
...
+                    </ul>
+                </div>
+            </section>
+        
+            <!--About Me Section-->
+            <section class="about">
+                <div class="about-me">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button" class="btn">Get My Resume</button>
```
Comment (ichala) [inline index.html]: - [x] kindly add the correct color for this button and try to apply the button styling: Active, Hover, and Pressed like shown in Figma

image

File: index.html [hunk 3]
```diff
@@ -1,65 +1,247 @@
...
+                </div>
+                <hr>
```
Comment (ichala) [inline index.html:170]: - [x]  kindly add the margins between the line and the background to match the Figma design


image

File: index.html [hunk 4]
```diff
@@ -1,65 +1,247 @@
...
+                    <h3>Frameworks</h3>
+                    </div>
+                    <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                    </ul>
+                </div>
```
Comment (ichala) [inline index.html]: - [x] The technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in  a `ul`, and  each of them in a `li` , kindly fixt that

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @demix007, 

Good job so far! 👏👏👏 your website looks so awesome and clean, I'm impressed 💯 👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linters Errors ✔️
- Correct Readme Template ✔️
- Great Design✔️
- Clean Code ✔️

 ## Required Changes ♻️
 
- [x]  _**_Check The Comments Below_**_

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @demix007  , 

Great work on making the changes requested by a previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

- [Optional ] Kindly, try to fix the MIT LICENSE In README file, The Link is broken. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**